include/kernels/dslash_staggered.cuh

```diff
@@ -41,8 +41,11 @@ namespace quda
     const GL L; /** the long gauge field */
 
     const real a; /** xpay scale factor */
-    const real tboundary;
+    const real tboundary; /** temporal boundary condition */
+    const bool is_first_time_slice; /** are we on the first (global) time slice */
```

Comment:
@maddyscientist is there a reason you've added these variables if they're never used? (in contrast to in `index_helper.cuh`, where you did make use of them)

Reply:
Not sure I follow.  These variables are used in index_helpher.cuh, and this is where they are defined.

Reply:
Oh, yep, mental disconnect. In my mind they were also defined in `include/index_helper.cuh`.